<ui version="4.0" >
 <class>AddButtonDialog</class>
 <widget class="QWidget" name="AddButtonDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>185</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Add Button</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Standard Button</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" colspan="2" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Standard button:</string>
         </property>
         <property name="buddy" >
          <cstring>standardButtonCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QComboBox" name="standardButtonCombo" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="standardText" >
         <property name="text" >
          <string>Text:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2" >
        <widget class="QLabel" name="standardTextLabel" >
         <property name="text" >
          <string>test</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="standardRole" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Button role:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2" >
        <widget class="QLabel" name="standardRoleLabel" >
         <property name="text" >
          <string>test</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="customTab" >
      <attribute name="title" >
       <string>Custom Button</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Button role:</string>
         </property>
         <property name="buddy" >
          <cstring>customButtonCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QComboBox" name="customButtonCombo" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Text:</string>
         </property>
         <property name="buddy" >
          <cstring>customButtonLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="customButtonLineEdit" />
       </item>
       <item row="2" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>16</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="okCancelDialogBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
